lib/sdf-extract/src/change_set.rs

```diff
@@ -46,11 +49,48 @@ impl FromRequestParts<AppState> for ChangeSetDalContext {
         parts: &mut Parts,
         state: &AppState,
     ) -> Result<Self, Self::Rejection> {
+        // Get the workspace we are accessing (and authorized for)
+        let ChangeSetAuthorization { ctx, .. } = parts.extract_with_state(state).await?;
```

Comment:
This code has literally just moved down into ChangeSetAuthorization, which is a direct analogue of WorkspaceAuthorization.